tensorflow/compiler/mlir/lite/python/graphdef_to_tfl_flatbuffer.cc

```diff
@@ -142,6 +142,7 @@ Status ConvertGraphDefToTFLiteFlatBuffer(const toco::ModelFlags& model_flags,
   bool run_quantize = tensorflow::ShouldRunQuantizePasses(module.get());
   mlir::TFL::PassConfig pass_config{
     emit_builtin_tflite_ops : emit_builtin_tflite_ops,
```

Comment:
The failures on windows look related:

tensorflow/compiler/mlir/lite/python/graphdef_to_tfl_flatbuffer.cc(144): error C2059: syntax error: ':'
tensorflow/compiler/mlir/lite/python/graphdef_to_tfl_flatbuffer.cc(148): error C2143: syntax error: missing ';' before '}'
tensorflow/compiler/mlir/lite/python/graphdef_to_tfl_flatbuffer.cc(149): error C2065: 'pass_config': undeclared identifier
tensorflow/compiler/mlir/lite/python/graphdef_to_tfl_flatbuffer.cc(149): error C2065: 'pm': undeclared identifier
tensorflow/compiler/mlir/lite/python/graphdef_to_tfl_flatbuffer.cc(149): error C4430: missing type specifier - int assumed. Note: C++ does not support default-int
tensorflow/compiler/mlir/lite/python/graphdef_to_tfl_flatbuffer.cc(149): error C2365: 'tensorflow::AddTFToTFLConversionPasses': redefinition; previous definition was 'function'
.\tensorflow/compiler/mlir/lite/tf_tfl_passes.h(34): note: see declaration of 'tensorflow::AddTFToTFLConversionPasses'
tensorflow/compiler/mlir/lite/python/graphdef_to_tfl_flatbuffer.cc(149): error C2513: 'tensorflow::AddTFToTFLConversionPasses': no variable declared before '='
tensorflow/compiler/mlir/lite/python/graphdef_to_tfl_flatbuffer.cc(149): fatal error C1903: unable to recover from previous error(s); stopping compilation